R/ptype-abbr-full.R

```diff
@@ -32,15 +32,25 @@
 #' cat(vec_ptype_abbr(1:10))
 vec_ptype_full <- function(x, ...) {
   check_dots_empty0(...)
-  UseMethod("vec_ptype_full")
+
+  # Data frames dispatch to the `data.frame` method by default to get
+  # the inner types format
+  method <- s3_method_specific(
+    x,
+    "vec_ptype_full",
+    ns = "vctrs",
+    df_default = TRUE
```

Comment:
You use this for full but not abbr?

Reply:
yes the data.frame method was guarded by `inherits_only()` already.